domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -2903,50 +2904,27 @@ public void selectStream(AsyncResult<SQLConnection> conn, String sql, JsonArray
   /**
    * Execute an INSERT, UPDATE or DELETE statement.
    * @param sql - the sql to run
-   * @param replyHandler - the result handler with UpdateResult converted toString().
-   * @deprecated use {@link #execute(String, Handler)} instead.
+   * @param replyHandler - the result handler with UpdateResult
    */
-  @Deprecated
-  public void mutate(String sql, Handler<AsyncResult<String>> replyHandler) {
-    execute(sql, res -> {
-      if (res.failed()) {
-        replyHandler.handle(Future.failedFuture(res.cause()));
-        return;
-      }
-      replyHandler.handle(Future.succeededFuture(res.result().toString()));
-    });
+  public void execute(String sql, Handler<AsyncResult<RowSet<Row>>> replyHandler)  {
+    execute(sql, Tuple.tuple(), replyHandler);
   }
 
   /**
-   * Execute an INSERT, UPDATE or DELETE statement.
-   * @param sql - the sql to run
-   * @param replyHandler - the result handler with UpdateResult
+   * Get vertx-pg-client connection
+   * @param replyHandler
    */
-  public void execute(String sql, Handler<AsyncResult<UpdateResult>> replyHandler)  {
-    long s = System.nanoTime();
-    client.getConnection(res -> {
+  public void getConnection(Handler<AsyncResult<SqlConnection>> replyHandler) {
```

Comment:
Can getConnection return a PgConnection (not a SqlConnection)? https://github.com/eclipse-vertx/vertx-sql-client/blob/3.9.0/vertx-pg-client/src/main/java/io/vertx/pgclient/PgConnection.java#L34

Reply:
Done!

Reply:
Fine.